app/next-client-app/app/scanreports/[id]/tables/[tableId]/concept-tags.tsx

```diff
@@ -0,0 +1,43 @@
+import { deleteConcept } from "@/api/concepts";
+import { Badge } from "@/components/ui/badge";
+import { Button } from "@/components/ui/button";
+import { ApiError } from "@/lib/api/error";
+import { Cross2Icon } from "@radix-ui/react-icons";
+import { toast } from "sonner";
+
+export async function ConceptTags({ concepts }: { concepts: Concept[] }) {
+  const handleDelete = async (conceptId: number) => {
+    try {
+      await deleteConcept(conceptId);
+      toast.success("Concept Id Deleted");
+    } catch (error) {
+      const errorObj = JSON.parse((error as ApiError).message);
+      toast.error(
+        `Unable to delete Concept id from value Error: ${errorObj.detail}`,
+      );
+      console.error(error);
+    }
+  };
+
+  return concepts && concepts?.length > 0 ? (
+    <div className="flex flex-col items-start w-[250px]">
+      {concepts.map((concept) => (
+        <Badge className="bg-carrot hover:bg-carrot" key={concept.concept_code}>
```

Comment:
Badges have different colours depending on how they have been added, for example: https://ccom-dev.azurewebsites.net/scanreports/2560/tables/8903/fields/115672/

handled by this switch in the React code: https://github.com/Health-Informatics-UoN/CaRROT-Mapper/blob/feat/682/ar/sr-fields-api/app/react-client-app/src/components/ConceptTag.jsx#L14

You need to add something similar to handle V (vocab), R (reuse), M (manual)